core/templates/dev/head/css/oppia.css

```diff
@@ -4369,28 +4369,45 @@ div.oppia-issues-learner-action {
 }
 
 .oppia-issues-playthrough-modal-btn {
-  background: transparent;
-  border-radius: 0;
+  background-color: #009A8B;
   border: none;
+  border-radius: 15px;
+  box-shadow: 10px 0px #008080;
+  color: #fff;
+  cursor: pointer;
+  font-size: 24px;  
+  outline: none;
+  padding: 20px 20px;
+  text-align: center;
+}
+
+.oppia-issues-playthrough-modal-btn:hover {
+  background-color: #00b3a0;
+}
+
+.oppia-issues-playthrough-modal-btn:active {
+  background-color: #00b3a0;
+  box-shadow: 0 10px #008080;
+  transform: translateY(4px);
 }
 
 /* Values below were experimentally generated using 'http://csshexagon.com' */
+
```

Comment:
Remove extra new line.

Reply:
I don't know why but by removing the line some checks are giving error.
https://circleci.com/gh/oppia/oppia/32260?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link